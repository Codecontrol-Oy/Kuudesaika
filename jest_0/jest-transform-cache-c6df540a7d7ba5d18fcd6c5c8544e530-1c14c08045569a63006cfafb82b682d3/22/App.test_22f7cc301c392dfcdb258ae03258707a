'use strict';var _react = require('react');var _react2 = _interopRequireDefault(_react);
var _App = require('./App');var _App2 = _interopRequireDefault(_App);
var _sinon = require('sinon');var _sinon2 = _interopRequireDefault(_sinon);
var _reduxMockStore = require('redux-mock-store');var _reduxMockStore2 = _interopRequireDefault(_reduxMockStore);function _interopRequireDefault(obj) {return obj && obj.__esModule ? obj : { default: obj };}
var mockStore = (0, _reduxMockStore2.default)();
var dispatch = _sinon2.default.spy();

describe("<App />", function () {
    it("Matches snapshot", function () {
        var app = shallow(
        _react2.default.createElement(_App2.default, null));

        expect(app).toMatchSnapshot();
    });
});