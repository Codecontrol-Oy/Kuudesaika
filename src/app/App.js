import React, {Component} from 'react';
import theme from './theme/global.scss';
import Application from 'grommet/components/App';
import Header from 'grommet/components/Header';
import Section from 'grommet/components/Section';
import Anchor from 'grommet/components/Anchor';
import Paragraph from 'grommet/components/Paragraph';
import Heading from 'grommet/components/Heading';
import Footer from 'grommet/components/Footer';
import Box from 'grommet/components/Box';
import {Breadcrumb} from 'Components';
import {browserHistory} from 'react-router';

export default class App extends Component {
  constructor (props) {
    super(props);
  }

  render () {
    const location = browserHistory.getCurrentLocation();
    return (
      <Application centered={false}>
        <Header size={"large"} className={theme.header}>
          <Anchor className={theme.headerTitle} href={"https://6aika.fi/kaupungit"} target={"_blank"}>#6Aika</Anchor>
          <Heading align={"center"} strong tag={"h1"} uppercase>Decisions API React Client</Heading>
        </Header>
        <Section className={theme.app}>
          {location.pathname !== '/' && <Breadcrumb path={location} />}
          {this.props.children}
        </Section>
        <Footer size={"large"} className={theme.footer}>
        </Footer>
      </Application>
    );
  }
}