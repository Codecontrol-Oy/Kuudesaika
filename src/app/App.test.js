import React from 'react';
import App from './App';
import sinon from 'sinon';
import configureStore from 'redux-mock-store';
const mockStore = configureStore();
const dispatch = sinon.spy();

describe("<App />", () => {
    it("Matches snapshot", () => {
        const app = shallow(
            <App />
        );
        expect(app).toMatchSnapshot();
    });
}) 