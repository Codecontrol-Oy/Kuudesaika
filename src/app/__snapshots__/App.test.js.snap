// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Matches snapshot 1`] = `
<MuiThemeProvider>
  <App
    centered={false}
  >
    <Header
      align="center"
      direction="row"
      pad={
        Object {
          "between": "small",
          "horizontal": "none",
          "vertical": "none",
        }
      }
      responsive={false}
      size="large"
    >
      <Anchor
        animateIcon={true}
        href="/"
        method="push"
        tag="a"
      >
        #6Aika
      </Anchor>
      <Heading
        align="center"
        strong={true}
        tag="h1"
        uppercase={true}
      >
        Decisions API React Client
      </Heading>
    </Header>
    <Section
      pad={
        Object {
          "vertical": "medium",
        }
      }
    >
      <Breadcrumb
        path={
          Object {
            "action": "POP",
            "hash": "",
            "key": null,
            "pathname": "blank",
            "query": Object {},
            "search": "",
            "state": undefined,
          }
        }
      />
    </Section>
    <Footer
      align="center"
      direction="row"
      responsive={false}
      size="large"
    />
  </App>
</MuiThemeProvider>
`;
